pjmedia/src/pjmedia/vid_port.c

```diff
@@ -352,16 +352,23 @@ static struct fmt_prop find_closest_fmt(pj_uint32_t req_fmt_id,
 	/* Fill the nearest width list. */
 	if (diff_width1 <= diff_width2) {
 	    int k = 1;
```

Comment:
shouldn't k start from zero here?

Reply:
yes, it should :(, my mistake

Reply:
Should I make a new pull request with proper k value ?

Reply:
No, you shouldn't create a new PR. Just modify this branch and push the changes.